pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml

```diff
@@ -241,51 +241,6 @@ spec:
           workspace: release-workspace
       runAfter:
         - push-snapshot
-    - name: push-sbom-to-pyxis
```

Comment:
why are you removing this? not part of your FBC work I think

Reply:
PR description says it is just for testing and not merging